noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -39,18 +39,39 @@ global MAX_NOTE_ENCRYPTED_LOGS_PER_CALL: u32 = 16;
 global MAX_ENCRYPTED_LOGS_PER_CALL: u32 = 4; // If modifying, update DEPLOYER_CONTRACT_ADDRESS.
 global MAX_UNENCRYPTED_LOGS_PER_CALL: u32 = 4; // If modifying, update DEPLOYER_CONTRACT_ADDRESS.
 
```

Comment:
I had to re-order the constants here so that they get successfully evaluated in TS with the eval method --> the constants became interdependent